init.lua

```diff
@@ -239,7 +239,7 @@ vim.opt.rtp:prepend(lazypath)
 -- NOTE: Here is where you install your plugins.
 require('lazy').setup({
   -- NOTE: Plugins can be added with a link (or for a github repo: 'owner/repo' link).
-  'tpope/vim-sleuth', -- Detect tabstop and shiftwidth automatically
+  'nmac427/guess-indent.nvim', -- Detect tabstop and shiftwidth automatically
```

Comment:
```suggestion
  'NMAC427/guess-indent.nvim', -- Detect tabstop and shiftwidth automatically
```

nit: capitalize for consistency?

Reply:
I changed it.